### PR TITLE
[RELEASE-DO-NOT-Merge] Release 2022 04.2

### PR DESCRIPTION
**Pull request by raubitsj**
Fixes WB-NNNN
Fixes #NNNN

Description
-----------
What does the PR do?

Testing
-------
How was this PR tested?

Checklist
-------
- Name PR "[WB-NNNN][WB-MMMM] Add support for..." similar to entries in CHANGELOG.md
- Include reference to internal ticket "Fixes WB-NNNN" (and github issue "Fixes #NNNN" if applicable)

### CODE REVIEW

**Review by kptkin — APPROVED** [review]
🔥

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,8 +1,17 @@
+## 0.12.15 (April 21, 2022)
+
+#### :nail_care: Enhancement
+* Optimize wandb.Image logging when linked to an artifact by @annirudh in https://github.com/wandb/client/pull/3418
```
Comment (dmitryduev) [inline CHANGELOG.md:4]:
```suggestion
* Optimize wandb.Image logging when linked to an artifact by @tssweeney in https://github.com/wandb/client/pull/3418
```

File: functional_tests/artifacts/log-image-artifact-path.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import numpy as np
+import wandb
+
+
+def make_image():
+    return wandb.Image(np.random.randint(255, size=(32,32)))
```
Comment (dmitryduev) [inline functional_tests/artifacts/log-image-artifact-path.py:6]:
```suggestion
    return wandb.Image(np.random.randint(255, size=(32, 32)))
```

File: functional_tests/artifacts/log-image-artifact-path.yea [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    - path
+  - :op:not_contains_regex:
+    - :wandb:runs[1][files]
+    - media/images/image_(.*).png
+  - :op:not_contains:
+    - :wandb:runs[1][summary][image]
+    - path
+  - :op:contains:
+    - :wandb:runs[1][summary][image]
+    - artifact_path
```
Comment (dmitryduev) [inline functional_tests/artifacts/log-image-artifact-path.yea:20]:
```suggestion
    - artifact_path

```

**Review by dmitryduev — COMMENTED** [review]
Couple tiny things, otherwise LGTM.

File: CHANGELOG.md (hunk 1 above)
Comment (raubitsj) [reply CHANGELOG.md:4]: fixed

File: functional_tests/artifacts/log-image-artifact-path.py (hunk 2 above)
Comment (raubitsj) [reply functional_tests/artifacts/log-image-artifact-path.py:6]: lets fix this in master

File: functional_tests/artifacts/log-image-artifact-path.yea (hunk 3 above)
Comment (raubitsj) [reply functional_tests/artifacts/log-image-artifact-path.yea:20]: fix in master

**Pull request closed without merging**